ports/stm32/boards/B_L072Z_LRWAN1/README.md

```diff
@@ -0,0 +1,45 @@
+# B-L072Z-LRWAN1 Board Notes
+
+## LoRa on the B-L072Z-LRWAN1
+
+The [B-L072Z-LRWAN1 Discovery kit](https://www.st.com/en/evaluation-tools/b-l072z-lrwan1.html)
+can be used for applications involving LoRa communications. The heart of the
+board in a Murata CMWX1ZZABZ-091 module which contains an STM32L072CZ and a
+Semtech SX1276 LoRa transceiver.
+
+This MicroPython board definition is built to target the STM32L0 - and the
+micropython-lib [lora library](https://github.com/micropython/micropython-lib/tree/master/micropython/lora)
+provides LoRa functionality using the SX1276.
+
+### LoRa library installation
+
+See the [installation instructions](https://github.com/micropython/micropython-lib/tree/master/micropython/lora#installation)
+for the lora library. In short, `lora-sx127x` and either `lora-sync` or
+`lora-async` will need to be installed using `mpremote` and `mip`.
+
+```bash
+mpremote mip install lora-sync
```

Comment:
Sorry @mattytrentini, I didn't think this through fully when I replied to your comment last week. The B-L072Z is *really* low resource board, it has no `mip` support and no filesystem in the default build configuration (and only 40KB of free flash in that configuration.)

Once the `micropython-lib` submodule commit updates on master, suggest the best option will be to add a board `manifest.py` that freezes in these two packages. It'll eat up a good chunk of the remaining space, but most folks picking this board will be picking it for LoRa purposes so I think it makes sense to provide this in the default build. There probably aren't enough resources for `asyncio` plus a modem driver plus any useful firmware, so we don't need to worry about `lora-async`.

(Although to make anything useful that persists after a reset will still require freezing a user firmware, so maybe this part isn't even useful?)

I don't know if it's better to leave this PR open until the submodule bumps and then rebase it, or I can open a new PR based on this commit later once the submodule is bumped?

Reply:
No worries @projectgus! I had assumed it didn't have onboard `mip` support (hence why I used mip via mpremote) but I wasn't aware there was no filesystem; that is problematic. 😛 

Freezing code is the right solution - and I agree that it makes sense to freeze lora code on this board. 

I think closing this PR makes sense for now. It will remain linked in 10640 and can be referred to even when closed so we don't lose it.